zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -750,6 +750,35 @@ function NotebookCtrl($scope, $route, $routeParams, $location, $rootScope,
 
   $scope.savePermissions = function() {
     convertPermissionsToArray();
+    if ($scope.isOwnerEmpty()) {
+      BootstrapDialog.show({
+        closable: false,
+        title: 'warning Permissions',
```

Comment:
it's a bit odd if this is the display title of the dialog `warning Permissions`?
perhaps just `Setting Owners Permissions`?